pkg/util/version/version.go

```diff
@@ -32,7 +32,9 @@ func NewVersion(s string) (version *Version, err error) {
 		}
 	}
 	if len(parts) >= 3 {
-		version.Patch, err = strconv.Atoi(parts[2])
+		patch, err := strconv.Atoi(parts[2])
+		version.Patch = new(int)
```

Comment:
Why is this necessary? Warrants a comment. And should this happen after the error check?

Reply:
The error check should be before the Patch pointer allocation you are right, I'll add a comment.